ssl-config/build.gradle

```diff
@@ -1,10 +1,18 @@
 apply from: "${rootDir}/gradle/libs.gradle"
 apply from: "${rootDir}/gradle/publish.gradle"
 
+// declare dependency here so that it isn't inadvertently used
+def bcpkix = 'org.bouncycastle:bcpkix-jdk15on:1.54'
+
 dependencies {
     compile libs.guava
     compile libs.jackson.databind
 
+    // only use BouncyCastle libraries for compiling and testing
```

Comment:
under which conditions exactly do users need to declare a bouncycastle dependency? the readme ought to clarify this.